handout/a11y/key-concerns/semantic-markup.md

```diff
@@ -0,0 +1,158 @@
+# Semantic Markup
+
+## Using Proper HTML Elements and Attributes
+
+A common trap when structuring html is using too many divs and marking them up with classes or ids to indicate their role, for example:
+
+```
+@Component({
+  selector: 'ngc2-app'
+  template: `
+    <div class="header">
+      <div class="navigation">
+        <div class="item"><a [routerLink]="['']"><img src="https://angular.io/resources/images/logos/angular2/angular.svg" width="40"></a></div>
+        <div class="item"><a [routerLink]="['services']">Services</a></div>
+        <div class="item"><a [routerLink]="['process']">Process</a></div>
+        <div class="item"><a [routerLink]="['work']">Work</a></div>
```

Comment:
Could this example be extended to explicitly show that binding behaviour to a `div` when it could be tied to a `button` is incorrect?